src/migrate.c

```diff
@@ -316,6 +339,9 @@ void MigrateKeys(RedisRaftCtx *rr, RaftReq *req)
         goto exit;
     }
     req->r.migrate_keys.migrate_term = raft_get_current_term(rr->raft);
+    if (getMigrationSessionKey(rr, req, &req->r.migrate_keys.migration_session_key) != RR_OK) {
+        goto exit;
```

Comment:
Should we reply here?

Reply:
we should, I meant to include it in the function itself.  thought if better inside or out?

Reply:
I see you already added. That makes sense because you also reply on some other places inside MigrateKeys(), not inside another sub function. So, I guess now it is inline with the rest of the function.